invisible_flow/entities/data_allegationcategory.py

```diff
@@ -0,0 +1,8 @@
+from attr import dataclass
+
+
+@dataclass
+class AllegationCategory:
```

Comment:
Can we match the name of the file to match the case of the Class name? ie `data_allegation_category` ?

Reply:
We could do this, but it does not follow the convention already established by allegation, complainant, and investigator